Makefile

```
@@ -20,6 +20,7 @@ endif
 # 3. Otherwise set it to "development_build" by default, e.g. for developers running the Makefile locally.
 ROX_IMAGE_FLAVOR ?= $(shell if [[ "$(GOTAGS)" == *"$(RELEASE_GOTAGS)"* ]]; then echo "stackrox.io"; else echo "development_build"; fi)
 
+ROX_PRODUCT_BRANDING := STACKROX_BRANDING
```

I think this would overwrite the value set by environmental variable set with `make thing ROX_PRODUCT_BRANDING=xxx`.

I think what would help could be this

```suggestion
ROX_PRODUCT_BRANDING ?= STACKROX_BRANDING
```

---

Additionally, we need to make sure that the downstream build will get a different value here - is there a ticket for it already?